src/frontend-samples/snow-decoration-sample/README.md

```diff
@@ -0,0 +1,28 @@
+# Particle Effect - Snow & Rain
+
+Copyright © Bentley Systems, Incorporated. All rights reserved.
+
+This sample shows how to create particle effects using snow and rain as examples.
+
+## Purpose
+
+The purpose of this sample is to demonstrate the following:
+
+* Creating a decorator and rendering it in the active view.
+* Working with particle effects using the [ParticleCollectionBuilder](https://www.itwinjs.org/reference/imodeljs-frontend/rendering/particlecollectionbuilder) API.
+
+## Description
+
+This sample provides an example of a particle effect in view space (as opposed to world space).  The sample provides features to configure a decorator emitting snow particles.  The decorator is creating using predefined props defined in [SnowDecorationApp.ts](./SnowDecorationApp.tsx).  Select which effect is used with the drop down menu.
+
+For a basic decorator, the [Decorator](https://www.itwinjs.org/reference/imodeljs-frontend/views/decorator/) interface is implemented. The decorate method adds the [RenderGraphics](https://www.itwinjs.org/reference/imodeljs-frontend/rendering/rendergraphic/) to the given [DecorateContext](https://www.itwinjs.org/reference/imodeljs-frontend/rendering/decoratecontext). Once the decorator is created, pass it to the [ViewManager.addDecorator](https://www.itwinjs.org/reference/imodeljs-frontend/views/viewmanager/adddecorator/) method to have it rendered in all active views. This sample is specifically for working with particle effects. For examples of other decorators, see [Heatmap Decorator Sample](https://www.itwinjs.org/sample-showcase/?group=Viewer+Features&sample=heatmap-decorator-sample&imodel=Metrostation+Sample) and this sample's [HighlightEmitter](./FireDecorator.ts).  For another sample using particles in a decorator, see [Particle Decoration (Fire)](https://www.itwinjs.org/sample-showcase/?group=Viewer+Features&sample=fire-sample).
```

Comment:
The reference to Highlight Emitter should be removed.

In my opinion, this works better for linking to a different sample: [volume](../volume-query-sample/readme.md).  I like it since it doesn't reload the whole page.